### PR TITLE
NISP-1557: Change providing evidence message; move text to messages; …

### PR DESCRIPTION
**Pull request by fred-jackson**
@swatcats143 - I have created a new section for the 'gaps in your record' messaging. Removed some text from the page and moved it into the messages. Removed some HTML from the messages and moved it into the page. Changed the providing evidence message from contacting HMRC to a postal address.

### CODE REVIEW

File: app/uk/gov/hmrc/nisp/views/nirecordGapsAndHowToCheckThem.scala.html [hunk 1]
```diff
@@ -54,11 +54,11 @@ <h2 class="heading-medium gapsinyourrecord">@Messages("nisp.nirecord.gapsinyourr
 <h3>@Messages("nisp.nirecord.gapsinyourrecord.selfemployment.title")</h3>
 <p>@Messages("nisp.nirecord.gapsinyourrecord.selfemployment.desc")</p>
 <h2 class="heading-medium gapsinyourrecord">@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.title")</h2>
-<p>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.desc1")
-<a href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/national-insurance-deficiency-enquiries"  rel="external" target="_blank">@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.desc2")</a>.</p>
+<p>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.line1")</p>
+<p>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.addr1")</br>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.addr2")</br>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.addr3")</br>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.addr4")</p>
```
Comment (andrewgee) [inline app/uk/gov/hmrc/nisp/views/nirecordGapsAndHowToCheckThem.scala.html]: `</br>` should be `<br/>`

Comment (fred-jackson) [inline app/uk/gov/hmrc/nisp/views/nirecordGapsAndHowToCheckThem.scala.html]: So close 😐

On Wednesday, 6 January 2016, Andrew Gee <notifications@github.com> wrote:

> In app/uk/gov/hmrc/nisp/views/nirecordGapsAndHowToCheckThem.scala.html
> <https://github.com/hmrc/nisp-frontend/pull/11#discussion_r48977453>:
>
> > @@ -54,11 +54,11 @@ <h2 class="heading-medium gapsinyourrecord">@Messages("nisp.nirecord.gapsinyourr
> >  <h3>@Messages("nisp.nirecord.gapsinyourrecord.selfemployment.title")</h3>
> >  <p>@Messages("nisp.nirecord.gapsinyourrecord.selfemployment.desc")</p>
> >  <h2 class="heading-medium gapsinyourrecord">@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.title")</h2>
> > -<p>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.desc1")
> > -<a href="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/national-insurance-deficiency-enquiries"  rel="external" target="_blank">@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.desc2")</a>.</p>
> > +<p>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.line1")</p>
> > +<p>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.addr1")</br>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.addr2")</br>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.addr3")</br>@Messages("nisp.nirecord.gapsinyourrecord.ifyouhaveevidence.addr4")</p>
>
> </br> should be <br/>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hmrc/nisp-frontend/pull/11/files#r48977453>.
>


-- 
Fred Jackson | Content Designer | Digital Delivery Centre Newcastle | Room
BP9001 | Prudhoe House | Benton Park View | Newcastle upon Tyne | NE99 1ZZ
| fred.jackson@digital.hmrc.gov.uk

**Comment by fred-jackson** [comment]
@andrewgee - *bows head in shame* I have made the necessary changes to the code... I beg your forgiveness.

**Comment by andrewgee** [comment]
@fred-jackson you are forgiven! I can't say the same for @dspork though, he holds grudges.

**Comment by fred-jackson** [comment]
@swatcats143 and @InduTest - I have just added a minor change to the copy, as customers only need to provide copies of the necessary evidence... I'm pretty sure this is the last change. HONEST!!!

**Comment by dspork** [comment]
@andrewgee @fred-jackson I shall remember this.

**Pull request merged**